pkgs/development/python-modules/gpt4all/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "gpt4all";
+  version = "1.0.8";
+  format = "wheel";
+
+  disabled = pythonOlder "3.8";
+
+  src = let
+    system = {
+      "x86_64-linux" = {
+        name = "x86_64";
+        hash = "sha256-lrX44Tl4TZzhGuosttOdeGaUPXy1dJF69I7KTMyzGhI=";
+      };
+    }.${stdenv.system} or (throw "Unsupported system");
+  in fetchPypi {
+    inherit pname format version;
+    inherit (system) hash;
+    platform = "manylinux1_${system.name}";
+    dist = "py3";
+    python = "py3";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  propagatedBuildInputs = [
```

otherwise things will not work

https://github.com/nomic-ai/gpt4all/blob/main/gpt4all-bindings/python/setup.py#L76